src/test/serialization_tests.cpp

```diff
@@ -38,6 +38,8 @@ using KFSTensorProto = ::inference::ModelInferResponse::InferOutputTensor;
 
 using TFPredictRequest = tensorflow::serving::PredictRequest;
 using TFPredictResponse = tensorflow::serving::PredictResponse;
+using KFSPredictRequest = ::inference::ModelInferRequest;
```

Comment:
If KFS contains "serving" in the acronym, TF is missing the S

Reply:
Will fix in following MR